jujugui/static/gui/src/app/components/model-actions/model-actions.js

```diff
@@ -129,7 +129,7 @@ class ModelActions extends React.Component {
             </span>
           </span>
           <span className="model-actions__import model-actions__button"
-            onClick={!isReadOnly && this._handleImportClick.bind(this)}
+            onClick={isReadOnly ? undefined : this._handleImportClick.bind(this)}
```

Comment:
It seems a little odd that this logic is in here. I feel like it might be better placed in the handler function itself.